less/home-page.less

```diff
@@ -10,20 +10,34 @@
 
     p {
       padding:0 15px;
+      color: @shuttle-gray;
+      font-weight: 600;
     }
   
   }
 
   .space-images {
     display: flex;
+    justify-content: center;
```

Comment:
You could add a mixin that has the attributes for centre and flex and add them to components you need.